tree/dataframe/inc/ROOT/RDF/RInterface.hxx

```diff
@@ -1354,6 +1354,11 @@ public:
                                      "way to convert TTrees to RNTuple is through the RNTupleImporter.");
          }
 
+         if (!dirname.empty()) {
+            throw std::runtime_error(
+               "RDataFrame: Snapshotting an RNTuple to a TFile sub-directory is currently not supported.");
+         }
+
```

Comment:
See my other comment; I believe this may just work (with `TFile` and `RNTupleWriter::Apend`)

Reply:
Writing works, indeed. But there's no non-power user way currently to read these files back, so until there is I would prefer to disallow it.